pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -1027,14 +1027,20 @@ private QuestionRecord createRecordForQuestion(QuestionDef question, FormRespons
         if (question.isDeprecated() && !instance.hasAnswer(question.getStableId())) {
             return null;
         }
-        Answer answer = instance.getAnswer(question.getStableId());
+
+        final Answer answer = instance.getAnswer(question.getStableId());
         if (answer == null) {
             return null;
         }
-        QuestionType type = answer.getQuestionType();
+
+        final QuestionType type = answer.getQuestionType();
         if (type == QuestionType.DATE) {
             DateValue value = (DateValue) answer.getValue();
             return new DateQuestionRecord(question.getStableId(), value);
+        } else if (answer.getQuestionType() == QuestionType.NUMERIC) {
```

Comment:
else by default covers these Numeric & Decimal questions if no additional stuff needed and transforms to SimpleQuestionRecord.